guides/source/getting_started.md

```diff
@@ -400,7 +400,7 @@ error:
 ![Another routing error, uninitialized constant ArticlesController](images/getting_started/routing_error_no_controller.png)
 
 This error occurs because the route needs to have a controller defined in order
-to serve the request. The solution to this particular problem is simple: create
+to serve the request. The solution to this particular problem is to create
```

Comment:
Let's strike `particular` while we're at it. The context is supplied via `this`.